code/modules/hotspring/falling_leaves.dm

```diff
@@ -42,3 +42,29 @@
 	particle_effect = new(src, /particles/sakura, 6)
 	//render the steam over mobs and objects on the game plane
 	particle_effect.vis_flags &= ~VIS_INHERIT_PLANE
+
+/particles/maple
```

Comment:
uses space indentation, should be converted to tabs, you should not artificially add multiple spaces to line up code like this, just `color = "#E23B1E"`, *not* `color       = "#E23B1E"`